test/kubernetes/e2e/features/services/httproute/types.go

```diff
@@ -15,9 +15,12 @@ import (
 )
 
 var (
+	// This is fragile, but symlinks don't work when the tests are imported to another repo, and its better than duplicating the file
+	repoRoot = filepath.Join(util.MustGetThisDir(), "..", "..", "..", "..", "..", "..")
```

Comment:
https://github.com/solo-io/gloo/blob/d613a7d53616ab6b34144291bf4df69708728bb4/test/kubernetes/e2e/features/helm_settings/suite.go#L57 

Can we use GetModuleRoot instead? 

Alternatively, we could define a method in the crds/ package that returns the full path to a file, and then call it here. that way the functionalty lives in the package itself and others can use

Reply:
That does not work when imported into `solo-projects`

Reply:
hmm, and what I tried was `GetGoModule` not `GetModuleRoot`, so let me try that too

Reply:
If we exposed a method in the crds package that usesd "GetThisDir/file-name", couldn't that work?